example.env

```diff
@@ -1,3 +1,9 @@
 GO_ENV=dev
 BOT_TOKEN=
-CONFIG_PATH=configmaps
\ No newline at end of file
+CONFIG_PATH=configmaps
+QUIZ_BASE_URL=your_quiz_api_base_url
+QUIZ_TOKEN=your_quiz_api_token
+GUILD_ID=your_guild_id
+COMMUTER_ROLE_ID=your_commuter_role_id
```

Comment:
This can be added to the config map in the config repo, which we can search by name in other places